onnxruntime/python/session.py

```diff
@@ -206,6 +206,12 @@ def bind_output(self, name, device_type, device_id, element_type, shape, buffer_
                                                 device_id),
                                     element_type, shape, buffer_ptr)
 
+    def bind_output_name(self, name):
+        self._iobinding.bind_output_name(name)
+
+    def get_outputs(self):
+        return self._iobinding.get_outputs()
+
     def clear_binding_inputs(self):
```

Comment:
There needs to be clear documentation somewhere saying when/why you'd use this (and possibly for all the IOBinding stuff), preferably with a python example in the ORT python API docs.

e.g. we could start making an effort for this [https://github.com/microsoft/onnxruntime/blob/master/docs/python/api_summary.rst](https://github.com/microsoft/onnxruntime/blob/master/docs/python/api_summary.rst) to provide useful info and usage examples